templates/packages/flaghelp.html

```diff
@@ -24,7 +24,8 @@ <h3>Flagging Packages</h3>
         with your additional text.</p>
 
         <p><strong>Note:</strong> Please do <em>not</em> use this facility if the
-        package is broken! Use the <a target="_blank" href="https://bugs.archlinux.org/"
-            title="Arch Linux Bugtracker">bugtracker</a> instead.</p>
+        package is broken! File an issue on the package's <a target="_blank" href="https://gitlab.archlinux.org/archlinux/packaging/packages"
```

Comment:
I think this should also point the user to the correct GitLab repo directly, see https://github.com/archlinux/archweb/commit/fca3ff5166d227ed4474ea6804fa748bc572371a for the related changes.

Reply:
There is not enough context to make the link more specific. This template is rendered at https://archlinux.org/packages/flaghelp/ (i.e. it is not bound to a specific package).